Tools/cases_generator/analyzer.py

```diff
@@ -3,6 +3,7 @@
 import lexer
 import parser
 import re
+from lexer import *
```

Comment:
I personally don't like star imports. What do you think about explicit names? Or `import lexer` and then `lexer.NAME`?

Reply:
Oh sorry, I should have used a draft PR for now. I actually wanted the star import to see which names are needed and then I'll change them one by one.

Reply:
I decided to:

- avoid changing the current imports and other usage
- always use `import lexer as lx` when many constants need to be imported. Then, I use them as `lx.XXX`.